src/AUMCdb_MEDS/download.py

```diff
@@ -37,7 +39,7 @@ def download_data(
         >>> download_data(Path("data"), cfg, runner_fn=fake_shell_succeed)
         wget -r -N -c -np -nH --directory-prefix data http://example.com/dataset
         wget -r -N -c -np -nH --directory-prefix data http://example.com/common
-        >>> download_data(Path("data"), cfg, runner_fn=fake_shell_succeed, do_demo=True)
+        >>> download_data(Path("data"), cfg, runner_fn=fake_shell_succeed)
```

Comment:
@rvandewater could you please update these examples to reflect the behaviour of the AUMCdb pipeline?